codec/src/main/java/io/netty/handler/codec/compression/ZstdDecoder.java

```diff
@@ -15,14 +15,17 @@
  */
 package io.netty.handler.codec.compression;
 
-import com.github.luben.zstd.ZstdInputStreamNoFinalizer;
+import com.github.luben.zstd.BaseZstdBufferDecompressingStreamNoFinalizer;
+import com.github.luben.zstd.ZstdBufferDecompressingStream;
```

Comment:
you don't need this import - you can get the `recommendedTargetBufferSize` from `BaseZstdBufferDecompressingStreamNoFinalizer`

Reply:
hmm doesn't seem so.

Reply:
`ZstdBufferDecompressingStreamNoFinalizer` works tho

Reply:
aaa, right. Don't know why I had that `Base` preprended